101-bundle-json.md

```diff
@@ -70,6 +71,23 @@ The following is an example of a `bundle.json` for a bundled distributed as a _t
     }
   ],
   "name": "helloworld",
+  "outputs": {
+    "clientCert" : {
+      "path" : "/cnab/app/outputs/clientCert",
+      "sensitive" : true,
+      "type" : "file"
+    },
+    "hostName" : {
+        "apply-to" : ["install"],
```

Comment:
We should really fix `applyTo` here and also in the params section. It's the only one that does not use camel case.

Reply:
Can defer to later change, since it would be a compat breaker

Reply:
@jeremyrickard still `apply-to` here